scan/src/pages/BlockHomePage.re

```diff
@@ -1,10 +1,12 @@
 module Styles = {
   open Css;
 
-  let vFlex = style([display(`flex), flexDirection(`row), alignItems(`center)]);
+  let vFlex = align => style([display(`flex), flexDirection(`row), alignItems(align)]);
 
   let pageContainer = style([paddingTop(`px(50))]);
```

Comment:
The height of each row is too narrow when compare with design?

Reply:
done